hooks/generic-webhook/hook/hook.js

```diff
@@ -8,13 +8,17 @@ async function handle({
   webhookUrl = process.env["WEBHOOK_URL"],
   webhookUser = process.env["WEBHOOK_USER"],
   webhookPassword = process.env["WEBHOOK_PASSWORD"],
+  webhookApikey = process.env["WEBHOOK_APIKEY"],
+  webhookApikeySecret = process.env["WEBHOOK_APIKEY_SECRET"],
   axios = require('axios')
 }) {
   const findings = await getFindings();
 
   console.log(`Sending ${findings.length} findings to ${webhookUrl}`);
 
-  if (webhookUser && webhookPassword){
+  if (webhookApikey && webhookApikeySecret){
+    await axios.post(webhookUrl, {scan, findings }, {headers: { webhookApikey: webhookApikeySecret}});
```

Comment:
I don't think this works as intended as the key would always be hardcoded to `webhookApikey` not the value of the var.

I think it should work like this.

```suggestion
    await axios.post(webhookUrl, {scan, findings }, {headers: { [webhookApikey]: webhookApikeySecret}});
```

Reply:
@J12934  
In axios headers are implemented this way.

```javascript

const result = await axios.post('https://testapi.org/post', { name: 'John Doe' }, {
    headers: {
    'content-type': 'text/json'
    }
});

```
### Reference
https://stackabuse.com/how-to-send-headers-with-an-axios-post-request/

Reply:
@renatosazup yes, the problem is that you don't set the key properly but always send along the string "webhookApikey" along as the header name because you are not using computed property names but just a string there: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#computed_property_names